kv/mdbx/kv_mdbx.go

```diff
@@ -368,6 +371,12 @@ func (opts MdbxOpts) Open() (kv.RwDB, error) {
 		}
 
 	}
+	if !opts.inMem {
```

Comment:
- maybe for in-mem case also can add/remove
- create helper funcs: PathDbMap, addDBPath, deleteDBPath

Reply:
done